enterprise/server/remote_execution/container/container.go

```diff
@@ -250,11 +250,14 @@ type UsageStats struct {
 // that TaskStats() can just return a view of the data rather than requiring a
 // full re-encoding.
 type timelineState struct {
```

Comment:
maybe document the units for the fields in this struct

Reply:
Renamed the fields to include units